custom_components/mcp23017/binary_sensor.py

```diff
@@ -70,24 +80,35 @@ def int_config(pins):
         return gpinten
 
     def update_sensors(port):
-        for pin_num in mcp.int_flag:
-            _LOGGER.debug("Dispatching MCP23017: {} on port: {} ".format(
-                           DEVICE.format(i2c_address, pin_num),port))
-            dispatcher_send(hass, DEVICE.format(i2c_address, pin_num))
-        mcp.clear_ints()
+        # TODO: port is probably a bad name, most likely this can be the interrupt pin?
```

Comment:
agreed?

Reply:
I used ```port``` because that's the name being used in the Home Assistant raspberry pi based sensors and some other documentation but I agree, it's probably a bad name and ```interrupt_pin``` is a better name.